### DISCUSSION TITLE
Improve feature request/issue templates

### DISCUSSION THREAD

**Discussion by pmiossec** [gitextensions/gitextensions#9527] [General] [answered]
Github introduced (still in beta but to not loose the link and the idea) a new way to create the issues templates that allows more complex forms with some validation :

https://dev.to/github/how-to-build-google-like-forms-with-github-3ig2

Advantages :
* better UX
* validation so the user have to fill the required fields

Difficulties:
* check if could be filled with query parameters for issue reporting from GE
* transition from current form as current form will be still used by old versions of GE:
    * migrate 'feature request' form first and gather feedback 
    * being able to have the current and new form at the same time?

**Comment by RussKie** [comment, marked as answer]
No real issues, done in https://github.com/gitextensions/gitextensions/pull/9638.
I updated only one template, you're welcome to give a go to the remaining 😉
